### PR TITLE
Nav restructuring

### PR DESCRIPTION
**Pull request by tiffanyqi**
plus styling
Screen Shot 2023-09-29 at 11 23 36 AM

### CODE REVIEW

File: pages/overrides.scss [hunk 1]
```diff
@@ -142,14 +143,70 @@ $border-radius-lg: 100px;
...
+
+        // remove the vertical rule within dropdowns
+        .before\:nx-w-px:before {
+            width: 0;
+        }
+
+        .hover\:nx-bg-gray-100:hover {
+            background-color: colors.$purple20;
+            color: colors.$purple140;
+        }
```
Comment (tiffanyqi) [inline pages/overrides.scss]: forgot to comment this, but this was to modify the hover state

File: pages/overrides.scss [hunk 2]
```diff
@@ -142,14 +143,70 @@ $border-radius-lg: 100px;
...
+        li {
+            margin-left: 4px;
+            margin-right: 4px;
+        }
+
+        .nx-pt-1 {
+            padding-bottom: 0.25rem;
+        }
+    }
+
```
Comment (tiffanyqi) [inline pages/overrides.scss]: these modified the focus states

File: pages/overrides.scss [hunk 3]
```diff
@@ -142,14 +143,70 @@ $border-radius-lg: 100px;
...
+            color: colors.$purple140;
+        }
+
+        // modify the non-focused nav items
+        .nx-text-gray-500 {
+            color: colors.$purple200;
+        }
+
+        // bold the parent links
+        .nextra-scrollbar >.nx-transform-gpu > .nx-transition-opacity > .nx-flex > li > button,
```
Comment (eerikson) [inline pages/overrides.scss]: I like what you've had here with these immediate descendant selectors—they at least give us some idea in the future about what the intent was if something breaks.

Is there any equivalent we can do above and below, for example on line 192 and  153?

File: theme.config.tsx [hunk 4]
```diff
@@ -52,7 +53,38 @@ const config: DocsThemeConfig = {
   },
   primaryHue: { dark: 256, light: 259 },
   sidebar: {
+    defaultMenuCollapseLevel: 1,
     toggleButton: true,
+    titleComponent: ({ title, type }) => {
+      if (type === `separator`) {
+        let icon;
+        switch (title) {
+          case `INTRO`:
```
Comment (eerikson) [inline theme.config.tsx:62]: Is there any way we can infer these titles from elsewhere? Not a big deal if not.

**Review by eerikson — APPROVED** [review]
This is WILD, thanks so much for taking care of this. Just left a few quesitons/nits that I think shouldn't block you.

File: theme.config.tsx (hunk 4 above)
Comment (tiffanyqi) [reply theme.config.tsx:62]: they come from a `_meta.json` file... but I'm not sure we can share variables between the two

File: pages/overrides.scss (hunk 3 above)
Comment (tiffanyqi) [reply pages/overrides.scss]: will try!

**Comment by tiffanyqi** [comment]
Closing in favor of https://github.com/mixpanel/docs/pull/582

**Pull request closed without merging**